esp/services/esdl_svc_engine/esdl_binding.cpp

```diff
@@ -771,7 +771,7 @@ EsdlBindingImpl::EsdlBindingImpl(IPropertyTree* cfg,
         if (!m_bndCfg)
             DBGLOG("ESDL Binding: Warning could not fetch ESDL bind information for this ESP Binding %s", binding);
 
-        m_esdl.setown(createEsdlDefinition());
+        m_esdl.setown(createNewEsdlDefinition());
```

Comment:
createNewEsdlDefinition may never have been used and looks to me like it may cause a leak (double linking the pointer).. should fix it in this pull request.